README.md

```diff
@@ -1,2 +1,27 @@
-# openshift-is
```

Comment:
Why these lines are removed?

Reply:
Fix in https://github.com/wso2/openshift-is/pull/1/commits/86e40fc1e706c2890aacee75f5fec80a41ac98a1 commit